src/components/KPagination/KPagination.vue

```diff
@@ -331,15 +390,35 @@ watch(() => props.currentPage, (newVal, oldVal) => {
 // recalc if the total number of items changed (which changed pageCount)
 watch(pageCount, (newVal, oldVal) => {
   if (newVal !== oldVal) {
+    forwardDisabled.value = currPage.value === newVal
+    lastDetached.value = newVal > (sequentialItemsVisible.value + 2) + (2 * fittingNeighbors.value)
+
     pagesVisible.value = getVisiblePages(
       currentlySelectedPage.value,
       newVal,
       false,
-      newVal > 5 + 2 * props.neighbors,
+      lastDetached.value,
     )
+  }
+})
 
-    forwardDisabled.value = currPage.value === newVal
-    lastDetached.value = newVal > 5 + 2 * props.neighbors
+watch(pagesVisible, () => {
+  if (!props.disablePageJump && !props.offset) {
+    fixOverflow()
```

Comment:
issue: if I scale the page down, the number of visible page jump items decreases; however, if I scale the page back up, the discarded items do not "come back"